### PR TITLE
Added a text under the title

### PR DESCRIPTION
**Pull request by Ahmad-Ayyoub**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -17,6 +17,8 @@
 :source-highlighter: prettify
 :figure-caption!:
 = Injecting dependencies into microservices
+[.hidden]
```
Comment (evelinec) [inline README.adoc]: Can you add a space here to make it clearer. Thanks

Comment (Ahmad-Ayyoub) [reply README.adoc]: Sure I will

**Comment by yeekangc** [comment]
@Ahmad-Ayyoub, please put in a meaningful description for this PR.

**Comment by Ahmad-Ayyoub** [comment]
Add link to published website from repo

**Pull request merged**